src/components/portrait/portrait-gravatar.d.ts

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
```

Comment:
This file is not needed

Reply:
I would not remove this file, as it might be useful if/when we'll decide to implement TypeScript in Carbon